From 19412dcdf1a9e868566f8030324b1ab903ca0a34 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Fri, 15 Feb 2019 21:31:14 +0100
Subject: [PATCH] CMake: Reorganize files

1. Rename src/config.vala.cmake to src/config.vala.in

2. Move com.github.parnold-x.nasc.gschema.xml to data subdirectory
---
 CMakeLists.txt                                                | 4 ++--
 .../com.github.parnold-x.nasc.gschema.xml                     | 0
 src/{config.vala.cmake => config.vala.in}                     | 0
 3 files changed, 2 insertions(+), 2 deletions(-)
 rename com.github.parnold-x.nasc.gschema.xml => data/com.github.parnold-x.nasc.gschema.xml (100%)
 rename src/{config.vala.cmake => config.vala.in} (100%)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eecfb32..c24fd09 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,11 +30,11 @@ list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 file(GLOB_RECURSE sources src/*.vala)
 
 # Some configuration
-configure_file (${CMAKE_SOURCE_DIR}/src/config.vala.cmake ${CMAKE_SOURCE_DIR}/src/config.vala)
+configure_file (${CMAKE_SOURCE_DIR}/src/config.vala.in ${CMAKE_SOURCE_DIR}/src/config.vala)
 
 #GSettings
 include(GSettings)
-add_schema("com.github.parnold-x.nasc.gschema.xml")
+add_schema("data/com.github.parnold-x.nasc.gschema.xml")
 
 set(NEEDED_PKG
     glib-2.0
diff --git a/com.github.parnold-x.nasc.gschema.xml b/data/com.github.parnold-x.nasc.gschema.xml
similarity index 100%
rename from com.github.parnold-x.nasc.gschema.xml
rename to data/com.github.parnold-x.nasc.gschema.xml
diff --git a/src/config.vala.cmake b/src/config.vala.in
similarity index 100%
rename from src/config.vala.cmake
rename to src/config.vala.in
